test/common.py

```diff
@@ -440,18 +485,16 @@ def test_scoring(data: Dict[str, Any], translate_params: str, test_similar_score
     with open(out_path) as score_out:
         score_scores = [float(line.strip()) for line in score_out]
 
-    # Compare scored output to original translation output. There are a few tricks: for blank source sentences,
-    # inference will report a score of -inf, so skip these. Second, we don't know if the scores include the
-    # generation of </s> and have had length normalization applied. So, skip all sentences that are as long
-    # as the maximum length, in order to safely exclude them.
+    # Compare scored output to original translation output. Unfortunately, sockeye.translate doesn't enforce
+    # that </s> was generated, and we can't know outside the decoder. So we use scores from force-decoding
```

Comment:
with the recent PR that disabled scoring tests for integration tests (models which have only received 1 or 2 updates) scoring system tests as they were should pass just fine.